sys/parsoid.js

```diff
@@ -417,6 +439,18 @@ PSP.getFormat = function(format, restbase, req) {
     });
 
     if (req.headers && /no-cache/i.test(req.headers['cache-control'])) {
+        if (!self._okayToRerender(req)) {
+            // Still update the revision metadata.
+            self.getRevisionInfo(restbase, req);
+            throw new rbUtil.HTTPError({
```

Comment:
No, the only thing cancelling the parallel revInfo request would be the process exiting, or some failure inside the revInfo request.